### ISSUE TITLE
docs build is failing

### ISSUE THREAD

**Issue by chrishavlin** [yt-project/yt_idv#119]
sphinx version needs updating, latest build has:

```
Sphinx version error:
The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.
```

**Comment by chrishavlin** [comment]
huh. that's actually a good question, i didn't look too closely at what exactly needed a higher Sphinx version -- maybe we can just disable that instead

**Comment by chrishavlin** [comment]
https://github.com/sphinx-doc/sphinx/issues/11890 updating the sphinx version fixed the issue in #124
